code/src/components/Header.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { tasks } from '../reducers/tasks'
+import { useSelector } from 'react-redux'
+import './header.css'
+
+export const Header = () => {
+
+const quantity = useSelector((state) => state.tasks.items )
+const totalQuantity = quantity.length
```

Comment:
Just select the length from the global store.
`const quantity = useSelector((state) => state.tasks.items.length )`